presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1113,6 +1114,15 @@ public Void visitTableWriter(TableWriterNode node, Integer indent)
             return processChildren(node, indent + 1);
         }
 
+        @Override
+        public Void visitAnalyzeFinish(AnalyzeFinishNode node, Integer indent)
+        {
+            print(indent, "- AnalyzeCommit[%s] => [%s]", node.getTarget(), formatOutputs(node.getOutputSymbols()));
+            printPlanNodesStatsAndCost(indent + 2, node);
```

Comment:
nit: it doesn't make much sense to show stats for this node

Reply:
I prefer to keep it for:
(1) consistency across other operators. 
(2) debugging uses. Connectors can specify customize operation in the `finishAnalyze` operator which may cost extra CPU.